common/src/address/hexified.rs

```diff
@@ -337,47 +338,6 @@ mod tests {
         assert_eq!(res, "0xabcd");
     }
 
-    #[test]
-    fn invalid_match_should_replace_with_hex_creating_case_address_creation_error() {
```

Comment:
Like discussed in the daily, see if you can restore these two tests with other cases that test the same thing and arrive to the same result.